semeio/jobs/correlated_observations_scaling/obs_utils.py

```diff
@@ -42,6 +43,16 @@ def find_and_expand_wildcards(obs_list, user_dict):
                     new_entries.append(val)
             new_dict[main_key]["keys"] = new_entries
 
+        # CALCULATE_KEYS.keys went from having some values, to having no values
+        if (
+            main_key == "CALCULATE_KEYS"
+            and len(value["keys"]) > 0
+            and len(new_entries) == 0
+        ):
+            raise ValidationError(
+                "Invalid configuration",
+                ["CALCULATE_KEYS {} matched no keys".format(value["keys"])],
```

Comment:
What if I have some valid and some invalid keys, then this will tell me all of them are invalid?

Reply:
Think it would be good to add a test for that, to check that only the invalid keys are in the error message.

Reply:
I've added a case for this (I think), but if there's a non-wildcard key there, this will not be thrown. If there's a wildcard in there that will match, this will not be thrown.

Let me know if there are additional things you wanted to test.

Reply:
I think maybe what @oyvindeide is referring to here is a case where one of the wildcard keys has a match and another key does not? I would be willing to argue that this should also be considered invalid and then adding the test @oyvindeide refers to would make sense. Am I correct Øyvind?

But that could potentially be resolved in a separate PR also...

Reply:
Both, in the first iteration any valid and invalid key would give both as invalid. But I have no problem with creating an issue for the remaining 👍

Reply:
Yes, so then you would expect the keys `FOPT, TYPO*` to yield an error right?